core/src/androidTest/java/com/cloudinary/android/AndroidJobStrategyTest.java

```diff
@@ -23,14 +22,14 @@ public void testAdapter() throws InterruptedException, IOException {
         int tenMinutes = 10 * 60 * 1000;
         UploadRequest<FilePayload> request = buildUploadRequest(payload, tenMinutes);
 
-        JobRequest adapted = AndroidJobStrategy.adapt(request);
+        WorkRequest adapted = AndroidJobStrategy.adapt(request);
 
-        Assert.assertEquals(20, adapted.getStartMs());
-        Assert.assertEquals(tenMinutes, adapted.getEndMs());
-        Assert.assertEquals(true, adapted.requiresCharging());
-        Assert.assertEquals(false, adapted.requiresDeviceIdle());
-        Assert.assertEquals(100, adapted.getBackoffMs());
-        Assert.assertEquals(JobRequest.BackoffPolicy.LINEAR, adapted.getBackoffPolicy());
-        Assert.assertEquals(9, adapted.getExtras().get("maxErrorRetries"));
+        Assert.assertNotNull(adapted.getId());
+//        Assert.assertEquals(tenMinutes, adapted.getEndMs());
```

Comment:
Please update or just remove these tests, if they are not relevant

Reply:
Fixed

Reply:
[EverNote](https://github.com/evernote/android-job) package is used as a work manager for async threads and processes it is deprecated and causes a whole lot of warnings to be thrown when used.
I've removed the package from the SDK completely and replaced the entire AndroidJobStrategy with Google's Android [WorkManager](https://developer.android.com/topic/libraries/architecture/workmanager?gclid=Cj0KCQjw94WZBhDtARIsAKxWG-_gzegkYaUWHbm8SvxHUj9AlQtVBa7Cqf-LBP7Dk_5k95SO1nTxGKAaAsa-EALw_wcB&gclsrc=aw.ds) which is recommended by Google and EverNote.